proto/store/slow_query.proto

```diff
@@ -0,0 +1,44 @@
+syntax = "proto3";
+
+package bytebase.store;
+
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/duration.proto";
+
+option go_package = "generated-go/store";
+
+// SlowQueryStatistics is a summary of slow queries.
+message SlowQueryStatistics {
+  // sql_fingerprint is the fingerprint of the slow query.
+  string sql_fingerprint = 1;
+  
+  // count is the number of slow queries with the same fingerprint.
+  int32 count = 2;
+  
+  // latest_log_time is the time of the latest slow query with the same fingerprint.
+  google.protobuf.Timestamp latest_log_time = 3;
+  
+  // details is the details of the slow query.
+  repeated SlowQueryDetails details = 4;
```

Comment:
nit: is it samples?

Reply:
Yes, we only store part of the details information